Public/Src/Cache/ContentStore/Distributed/NuCache/ContentLocationDatabase.cs

```diff
@@ -77,6 +77,8 @@ public abstract class ContentLocationDatabase : StartupShutdownSlimBase
         /// </summary>
         private long _cacheUpdatesSinceLastFlush = 0;
 
+        internal long CacheUpdatesSinceLastFlush => _cacheUpdatesSinceLastFlush;

```

Comment:
nit: nodoc or a small comment that explains that this thing is used by tests only.